Documentation/operations/upgrade.rst

```diff
@@ -318,6 +318,22 @@ Annotations:
   applications create new connections after the TTL specified by the upstream
   DNS server is expired.
 
+New Options
+~~~~~~~~~~~
+
+* ``routing-mode=native``: This option enables native-routing mode, in place of
+  ``tunnel=disabled``, now deprecated.
+* ``tunnel-protocol``: This option allows setting the tunneling protocol, in place
+  of e.g., ``tunnel=vxlan``.
+
+Deprecated Options
+~~~~~~~~~~~~~~~~~~
+
+* The ``tunnel`` option was deprecated and will be removed in v1.14. To enable
```

Comment:
Did you mean v1.15?

Reply:
Ouch, yes. That's because this is a rebase of a PR I already attempted to send into v1.14.